zoom_monitor.py

```diff
@@ -35,7 +37,16 @@ def __init__(self, url, log_level):
         # exits. It's a useful option when debugging or adding new features.
         #chrome_options.add_experimental_option("detach", True)
 
+        # Normally, if you hit control-C, Selenium shuts down the web browser
+        # immediately. However, we want to leave the meeting before
+        # disconnecting. So, we need to make the subprocess running the web
+        # browser be in a separate process group from ourselves, so it doesn't
+        # receive the signal.
```

Comment:
Be more specific about what signal we're talking about. It's the SIGINT that the OS sends when you press control-C.